.circleci/config.yml

```
@@ -84,6 +84,11 @@ jobs:
     steps:
       - checkout
       - install_terraform
+      - run:
```

probably should move `terraform fmt` before this since the check will fail fast